ui-tests/cypress/support/pageObjects/content/assets/AssetsPage.js

```diff
@@ -49,114 +85,87 @@ export default class AssetsPage extends AdminContent {
   }
 
   getFilterResultInfo() {
-    return this.getAssetsBody()
+    return this.get()
                .find(this.resultInfo);
   }
 
   getFilterResultItemCount() {
     return this.getFilterResultInfo()
+               .find(`#search > .pager`)
                .find(this.resultInfoItemCount);
   }
 
-  getTable() {
-    return this.getAssetsBody()
-               .find(htmlElements.table);
-  }
-
   getTableRows() {
-    return this.getTable()
-               .children(htmlElements.tbody)
-               .children(htmlElements.tr);
+    return this.getAssetsBody();
   }
 
   getKebabMenu(code) {
     return new AssetsKebabMenu(this, code);
   }
 
-  selectFiles(...fileName) {
-    this.getFileInput().selectFile(fileName, {force: true});
-    this.parent.getDialog().setBody(AddAssetDialog);
-  }
 }
 
 class AssetsKebabMenu extends KebabMenu {
```

Comment:
Is this _KebabMenu_ so different from `KebabMenu`?

Reply:
unfortunately, I have not found another way to interact with the actions buttons.

Reply:
What I meant is that, since you have overwritten the methods of `KebabMenu`, either `KebabMenu` needs to be updated to accomodate this object or this object is not a `KebabMenu`

Reply:
I think I have solved it.